modular_skyrat/modules/customization/modules/language/nekomimetic.dm

```diff
@@ -0,0 +1,3 @@
+// Nekomimetic, my beloved
+/datum/language/nekomimetic
+	secret = TRUE
```

Comment:
I'll miss Nekomimetic, maybe one day I'll find a way for it to make sense to come back, without being as meme-y (because god damn I hated that part of it).